src/ComputeSharp.D2D1/Shaders/Interop/D2D1ShaderCompilerOptions.cs

```diff
@@ -0,0 +1,172 @@
+﻿using System;
+using TerraFX.Interop.DirectX;
+
+namespace ComputeSharp.D2D1.Interop;
+
+/// <summary>
+/// Flags to control the options that can be used to compile a D2D1 shader.
+/// </summary>
+/// <remarks>
+/// For more info, see For more info, see <see href="https://docs.microsoft.com/windows/win32/direct3dhlsl/d3dcompile-constants"/>.
+/// </remarks>
+[Flags]
+public enum D2D1ShaderCompilerOptions
```

Comment:
Should also have a `None = 0` value, which is pretty standard

Reply:
I'm confused, where would that be useful though 🤔